charts/cronjob/templates/cronjob.yaml

```diff
@@ -9,6 +9,11 @@ metadata:
     heritage: {{ .Release.Service }}
 spec:
   schedule: {{ .Values.schedule | quote }}
+  {{- if .Values.runOnDemandOnly }}
+  suspend: true
+  {{ else if .Values.suspend }}
```

Comment:
why not use `or`? :) 
````go
{{- if .Values.runOnDemandOnly or .Values.suspend }}
````

Reply:
I wanted to use it but there are currently no `AND/OR/...` operators in helm charts - see flow documentation [here](https://docs.helm.sh/chart_template_guide/#flow-control).

I also tried your version:
```yaml
{{- if .Values.runOnDemandOnly or .Values.suspend }}
suspend: true
{{- end}}
```
And it gives me this error:
```
- Upgrading [on-demand-test] helm chart template with release name: [cronjob] on [staging] environment.
Error: UPGRADE FAILED: render error in "cronjob/templates/cronjob.yaml": template: cronjob/templates/cronjob.yaml:12:16: executing "cronjob/templates/cronjob.yaml" at <.Values.runOnDemandO...>: runOnDemandOnly is not a method but has arguments
```